jena-core/src/main/java/jena/RuleMap.java

```diff
@@ -50,7 +53,7 @@
  * </p>
  */
 public class RuleMap {
-    static { setLog4jConfiguration() ; }
+    static { setCmdLogging("jena-log4j.properties") ; }
     
```

Comment:
Sure, I'll make that change.I'm still getting my head (slowly) around the logging facilities.